src/foam/nanos/dao/Operation.js

```diff
@@ -21,28 +21,28 @@ foam.ENUM({
   values: [
     {
       name: 'CREATE',
-      label: 'Create',
+      label: { en: 'Create', pt: 'Crio'},
```

Comment:
If we include these here then everyone has to download the Portuguese text, even those that speak other languages. This makes the download larger for everyone and doesn't scale as we add support for more and more languages. Can't these just be defined in the locales journal?

Reply:
@kgrgreer should able to, will make the changes